examples/inshop/configs/train_inshop_arcface.yaml

```diff
@@ -0,0 +1,91 @@
+postfix: metric_learning
+
+seed: 42
+precision: 32
+accelerator: gpu
+devices: 1
+
+dataframe_name: df_no_bboxes.csv
+dataset_root: data/DeepFashion_InShop/
+logs_root: logs/DeepFashion_InShop/
+logs_folder: ${now:%Y-%m-%d_%H-%M-%S}_${postfix}
+
+num_workers: 20
+cache_size: 100000
+
+transforms_train:
+  name: augs_hypvit_torch
+  args:
+    im_size: 224
+
+transforms_val:
+  name: norm_resize_hypvit_torch
+  args:
+    im_size: 224
+    crop_size: 224
+
+sampler: null
+bs_train: 256
+
+bs_val: 256
+max_epochs: 10000
+valid_period: 1
+
+metric_args:
+  metrics_to_exclude_from_visualization: [cmc,]
+  cmc_top_k: [1]
+  map_top_k: [5]
+  return_only_main_category: True
+  visualize_only_main_category: True
+
+log_images: True
+
+metric_for_checkpointing: OVERALL/cmc/1
+
+model:
+  name: extractor_with_mlp
+  args:
+    mlp_features: [384, 256]
+    extractor:
+      name: vit
+      args:
+        normalise_features: False
+        use_multi_scale: False
+        weights: vits16_dino
+        arch: vits16
+
+map_labels: True
```

Comment:
it seems confusing

Reply:
moreover, we have map_labels, but don't have map_categories

Reply:
So what do you suggest?